packages/amplify-opensearch-simulator/package.json

```
@@ -0,0 +1,44 @@
+{
+    "name": "@aws-amplify/amplify-opensearch-simulator",
+    "version": "1.0.0",
+    "description": "Opensearch local simulator",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/aws-amplify/amplify-cli.git",
+        "directory": "packages/amplify-opensearch-simulator"
+    },
+    "author": "Amazon Web Services",
+    "license": "Apache-2.0",
+    "main": "index.js",
+    "keywords": [
+        "graphql",
+        "appsync",
+        "aws"
+    ],
+    "scripts": {
+        "test": "jest"
+    },
+    "dependencies": {
+        "amplify-cli-core": "3.3.0",
+        "aws-sdk": "^2.1113.0",
```

The sdk version here must match the rest of the CLI.

```suggestion
        "aws-sdk": "^2.1233.0",
```